tests/test_expression.py

```diff
@@ -729,7 +697,7 @@ def test_html_attrs(self):
         )
 
     @parametrize_context_behavior(["django", "isolated"])
-    def test_later_spreads_overwrite_earlier(self):
+    def test_later_spreads_do_not_overwrite_earlier(self):
```

Comment:
One of potentially breaking changes - Previously, when there was

`{% component ...attrs1 ...attr2 %}`

then if there was the same key in both `attrs1` and `attrs2`, the first one would be overwritten.

But as described in some of the other comments here, now this raises an error

Reply:
I think this is reasonable.